tensorflow_serving/model_servers/BUILD

```diff
@@ -141,6 +141,15 @@ py_test(
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.data-00000-of-00001",
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.index",
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.meta",
+        "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two2/00000123/export.data-00000-of-00001",
```

Comment:
Do you still need half_plus_two2? I was hoping we could just have:
 1. The original half_plus_two
 2. The new half_plus_three
(and nothing else)